Please accept me now!

philschulz opened the pull request.

Commit:
Add labels to definitions that indicate what is being defined.

Commit:
Address some of Christian's comments and make changes accordingly. Also update the distribution plot and add a cdf plot. Add some sentences about how one can see discreteness of a distribution in a cdf plot.

Commit:
Add section on negative binomial in chapter 3. Add contributors section to main script.

Commit:
Make further changes to chapter 3. Still need to fix lottery example.

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
+From the above definition, we see that the random variables need to
+share an underlying sample space. This also gives us more insight into
+the real power of RVs: in applications, we usually do not care at all
+about the underlying sample space but only about certain quantities
```
only about the quantities

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
+the real power of RVs: in applications, we usually do not care at all
+about the underlying sample space but only about certain quantities
+captured by the RVs. Therefore, we define a probability distribution over the
+random variables of interest. Since by definition the distribution has to fulfil conditions that would allow us to interpret it
```
"would"?
that allows us to interpret it

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
+captured by the RVs. Therefore, we define a probability distribution over the
+random variables of interest. Since by definition the distribution has to fulfil conditions that would allow us to interpret it
+as a probability measure on a probability space, we can use it without worrying
+about the underlying sample space too much. \chris{hmmm, not sure about this statement}\philip{I tried to formulate this a bit
```
"too much"?
sounds vague, we can leave it out.

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
 
-Now if we set $ X = (Z_{1}, \ldots, Z_{m}) $, we can recover $ P_{Z_{1}^{m}} $ from $ P_{YZ_{1}^{m}} $ in the same way. Likewise,
-if we set $ Y = (Z_{1}, \ldots, Z_{m}) $ we get the generalization of \ref{simpleMarginal}. We will assume that each $ Z_{j} $
+If we set $ X = (Z_{1}, \ldots, Z_{m}) $, we can recover $ P_{Z_{1}^{m}} $ from $ P_{Z_{1}^{m}Y} $ in the same way.\chris{so?}\philip{Sorry, I do not understand your comment here.} Likewise,
```
sorry for my too short comment, I somehow did not understand what this sentence adds in terms of scientific content. Is it such a common case that it needs to mention explicitly? It's totally obvious to me that you can do the same thing no matter what the RV X is.
Ah, I see that you are kind of using it as bridge to the other case where you set Y to be all these Z's...
still, I think we should skip the first part (setting X to Z_1^m)

philschulz replied:
Yes, it is obvious, but I am assuming here that people have never worked with random variables before and do not know that it is "allowed" to a assign a random vector to a new variable. That's why I would rather leave it in.

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -429,81 +460,125 @@ \section{Two important distributions}
-The Bernoulli is defined for one coin flip, or more generally for the drawing of a coloured ball from an urn containing balls
-of exactly two colours. Now what if we draw balls repeatedly with replacement? We will get a sequence of coloured balls. 
+The Bernoulli is defined for one coin flip, or more generally for the
+drawing of a coloured ball from an (very large) urn containing balls
```
a (very large) urn

cschaffner commented:
maybe the bars in the bar-plot could be a thicker? so that the thin red line does not cover them entirely?

cschaffner commented:
Thanks a lot for all the work, @philschulz . It looks pretty good to me on first sight!

Commit:
Fixed lottery example.

philschulz commented:
Hey Christian,

I will incorporate your comments and try to resolve the merge conflict either today or tomorrow. Making the bars in the plot bigger is no problem, I will just adjust the R script.

Have a nice weekend!

cschaffner commented:
@philschulz , I have some more changes that you can find in https://github.com/BasicProbability/LectureNotes/pull/15
I guess you will have to merge them manually again...

Commit:
Update line thickness in plots.

Commit:
Incorporate Christian's suggestions.

Commit:
Untrack pdfs in order to resolve merge conflicts.

philschulz commented:
Hey Christian,

I incorporated your proposed changes and removed the merge conflicts. The pull request can now be accepted.

Here is what remains to do: 

1) I have made a couple of inline comments myself, mostly in response to yours. If you regard the problems as resolved, just erase the comments. 

2) Make the usual chapter3.tex and chapter3_forInclude.tex partition. This should only be done AFTER this request has been accepted. 

3) Compile the pdfs after this request has been accepted.

4) Update script and website.

One last remark: Merge conflicts seem to ensue whenever we update the pdfs (that's also why the pdfs in this pull request have not been updated). For the future we should probably only update the remote pdfs once we have a final version.

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -198,11 +227,11 @@ \section{Expectation and Variance}
+\begin{Definition}[Expectation]
 The \textbf{expectation} of a random variable $ X $ with respect to the distribution $ P $ is defined as
-$$ \E[X] := \sum_{x \in \supp(X)} x P(X=x) $$ \, .
+$$ \E[X] := \sum_{x \in \supp(X)} x P(X=x) $$ \, . \philip{Are you sure you want to have this dot here? It seems to unnecessarily inflate the box.}
```
the dot should have been before the $$

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -379,15 +419,14 @@ \section{Joint and conditional distributions}
-themselves\chris{what do you mean?} as it is unclear what this would even be.
-What we can do is to compute the expectation of functions of them. The standard choice in the case of expectation
+to $P_{Y|X}=P_Y$). Moreover, independence makes it much easier to calculate the expectation and variance of jointly distributed random variables. We have to be careful, though. Jointly distributed random variables do not take on a joint value. Rather, each of the joint RVs still takes on its own value.\chris{what do you mean?}
+This is why in the joint case we have to compute the expectation of functions of the joint RVs.\philip{Is this better?} The standard choice in the case of expectation
```
I've inserted a foonote, see master

cschaffner commented on chapter3/chapter3.tex:
```diff
@@ -492,14 +529,14 @@ \section{Two\chris{or more?} Important Distributions}
+produce has a probability of being faulty. Obviously, you cannot sell faulty items to your customer. The Binomial distribution allows you to
+compute that probability that you will be able to meet the customer's demand. Based on this you can evaluate whether or not it is reasonable to
+accept the customer's order.\chris{I don't understand the example,
+  Bernoulli would be about \emph{counting} the number of failures.}\philip{Is this one better?}
```
much better!
